cpp/ql/src/Security/CWE/CWE-676/DangerousUseOfGets.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Use of dangerous function 'gets'
+ * @description The standard library 'gets' function is dangerous and should not be used.
```

Comment:
This new query is not in any suites, is it? We still use the hand-maintained suites for nightly jobs, and the security suites are used by some customers.

Reply:
Whoops, forgot to update suites as well.  Updated.